test/features/ExternalApi.test.ts

```diff
@@ -6,7 +6,9 @@ import * as vscode from "vscode";
 import { before, beforeEach, afterEach } from "mocha";
 import { IExternalPowerShellDetails, IPowerShellExtensionClient } from "../../src/features/ExternalApi";
 
-const testExtensionId = "ms-vscode.powershell-preview";
+// tslint:disable-next-line: no-var-requires
+const PackageJSON: any = require("../../../package.json");
```

Comment:
We should make sure that this still works when the extension is bundled to a VSIX (or possibly webpacked)

Reply:
It should, since this is just test code. Are the tests included in the VSIX?

Reply:
I don't think so, but I'm not sure. Possible to find out by decompressing the VSIX and inspecting it.

Reply:
LOL it's just a zip, who would have thought 🤦

Reply:
Yup, we're good!